aws_lambda_builders/utils.py

```diff
@@ -222,3 +223,32 @@ def extract_tarfile(tarfile_path: Union[str, os.PathLike], unpack_dir: Union[str
                 raise tarfile.ExtractError("Attempted Path Traversal in Tar File")
 
         tar.extractall(unpack_dir)
+
+
+def glob_copy(source: Union[str, list], destination: str) -> None:
+    items = source if isinstance(source, list) else [source]
+    dest_path = Path(destination)
+    known_paths = []
+    for item in items:
+        if Path(item).is_absolute():
+            raise ValueError('"{item}" is not a relative path'.format(item=item))
+        files = glob(item, recursive=True)
+        if len(files) == 0:
+            raise ValueError('"{item}" not found'.format(item=item))
+        for file in files:
+            save_to = str(dest_path.joinpath(file))
+            LOG.debug("Copying resource file from source (%s) to destination (%s)", file, save_to)
+            save_to_dir = os.path.dirname(save_to)
+            if save_to_dir not in known_paths:
+                os.makedirs(save_to_dir, exist_ok=True)
+                known_paths.append(save_to_dir)
+            shutil.copyfile(file, save_to)
+
+
+def get_option_from_args(args: Union[None, Dict[str, any]], option_name: str) -> any:
+    if args is not None:
+        if "options" in args:
+            if args["options"] is not None:
```

Comment:
Let's do 
```
if args.get("options", None) is not None: 
```
instead so that if options isn't there, it doesn't raise a key error. And the same thing for wherever else we do dictionary accesses like this

Reply:
Roger that.  Is this the correct way - should I be doing something more "pythonic"?

```python
def get_option_from_args(args: Union[None, Dict[str, any]], option_name: str) -> any:
    if args is not None:
        options = args.get("options", None)
        if options is not None:
            return options.get(option_name, None)
    return None
```